.github/workflows/deploy.yml

```diff
@@ -54,25 +54,21 @@ jobs:
 
       - name: Build HTML
         run: |
-          make html BUILD=foreman-el
-          make html BUILD=foreman-deb
-          make html BUILD=katello
-          make html BUILD=satellite
-          make html BUILD=orcharhino
+          make html BUILD=foreman-el LINKS=LOCAL
+          make html BUILD=foreman-deb LINKS=LOCAL
+          make html BUILD=katello LINKS=LOCAL
+          make html BUILD=satellite LINKS=LOCAL
+          make html BUILD=orcharhino LINKS=LOCAL
 
       - name: Upload HTML
         uses: actions/upload-artifact@v2
         with:
           name: foreman-docs-html-${{ env.BRANCH_NAME }}
           path: guides/build/
 
-      - name: Check HTML links
+      - name: Check HTML links of all builds
         run: |
-          make linkchecker-tryer BUILD=foreman-el
-          make linkchecker-tryer BUILD=foreman-deb
-          make linkchecker-tryer BUILD=katello
-          make linkchecker-tryer BUILD=satellite
-          make linkchecker-tryer BUILD=orcharhino
+          make linkchecker-tryer
```

Comment:
This was not necessary, this makefile target simply finds all HTML pages and checks them. We can only call this once, BUILD variable is unused for this case.